doc/classes/LinkButton.xml

```diff
@@ -26,7 +34,16 @@
 			Base text writing direction.
 		</member>
 		<member name="underline" type="int" setter="set_underline_mode" getter="get_underline_mode" enum="LinkButton.UnderlineMode" default="0">
-			Determines when to show the underline. See [enum UnderlineMode] for options.
+			The underline mode to use for the text. See [enum LinkButton.UnderlineMode] for the available modes.
+		</member>
+		<member name="url" type="String" setter="set_url" getter="get_url" default="&quot;&quot;">
+			The URL for this [LinkButton]. If a URL has been set, it will be opened automatically with the operating system's default program assigned to the URL protocol when the button is pressed. For HTTP and HTTPS URLs, this is typically the user's default Web browser. See [method OS.shell_open] for more information.
+			Examples
```

Comment:
```suggestion
			The URL for this [LinkButton]. If pressed and set, it opens the URL protocol's default program. HTTP and HTTPS URLs open the default web browser. See [method OS.shell_open] for more information.
			Examples:
```

I tried hard to shorten the description. I hope it is easier understandable now.